test/integration/rules/color-contrast/color-contrast.html

```diff
@@ -424,3 +424,13 @@
 <p style="color: #ccc; -webkit-text-stroke: 0.02em #444" id="fail12">
   Text-stroke 0.02em
 </p>
+
+<div
+  id="backdrop"
+  style="position: absolute; z-index: 1; height: 40px; background: #333"
```

Comment:
```suggestion
  style="position: absolute; z-index: 1; height: 40px; background: #333; width: 100%;"
```

Otherwise it's not visible, I noticed this was passing on develop.